### PR TITLE
wifivelocityd, wwand, xartstorageremoted, xsand: add page

### PR DESCRIPTION
**Pull request by drasbeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/osx/wifivelocityd.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# wifivelocityd
+
+> `launchd` daemon for the WiFiVelocity framework.
+> XPC helper for performing system context actions for the WiFiVelocity framework.
```
Comment (pixelcmtd) [inline pages/osx/wifivelocityd.md:4]:
```suggestion
> XPC helper for performing system context actions for the WiFiVelocity framework.
> This should not be invoked manually.
> More information: <http://www.manpagez.com/man/8/wifivelocityd/>.

- Start the daemon:

`wifivelocityd`
```

File: pages/osx/wwand.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# wwand
+
+> This is the USB WWAN device configuration daemon.
+> This should not be invoked manually.
```
Comment (pixelcmtd) [inline pages/osx/wwand.md:4]:
```suggestion
> This should not be invoked manually.
> More information: <https://keith.github.io/xcode-man-pages/wwand.8.html>.

- Start the daemon:

`wwand`
```

File: pages/osx/xartstorageremoted.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# xartstorageremoted
+
+> This is the xART Remote Storage Daemon.
+> It listens for save/fetch requests from the CoProcessor.
+> It must not be unloaded and should only be run via `launchd` when a request must be handled.
```
Comment (pixelcmtd) [inline pages/osx/xartstorageremoted.md:5]:
```suggestion
> xART Remote Storage Daemon. Receives save/fetch requests from the CoProcessor.
> This should not be invoked manually.
> More information: <http://www.manpagez.com/man/8/xartstorageremoted/>.

- Start the daemon:

`xartstorageremoted`
```

**Comment by drasbeck** [comment]
@pixelcmtd, thanks for the suggestions. I implemented them all, but had to revert wording in xartstorageremoted.md to conform to `TLDR003 Descriptions should start with a capital letter`.

File: pages/osx/xsand.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# xsand
+
+> Xsan file system management daemon.
+> Provides services for the Xsan file system.
+> It should not be invoked manually.
```
Comment (pixelcmtd) [inline pages/osx/xsand.md:5]: I'm really not happy about that link but there is hardly any official documentation

```suggestion
> Xsan file system management daemon. Provides services for the Xsan file system.
> It should not be invoked manually.
> More information: <https://developer.apple.com/support/downloads/Xsan-Management-Guide.pdf>.

- Start the daemon:

 `xsand`
```

File: pages/osx/wwand.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# wwand
+
+> This is the USB WWAN device configuration daemon.
```
Comment (kbdharun) [inline pages/osx/wwand.md:3]:
```suggestion
> USB WWAN device configuration daemon.
```
I think this would be better

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by drasbeck** [comment]
Thanks for the feedback. Will definitely make future PRs easier. 🦾

File: pages/osx/wifivelocityd.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# wifivelocityd
+
+> XPC helper for performing system context actions for the WiFiVelocity framework.
+> This should not be invoked manually.
```
Comment (navarroaxel) [inline pages/osx/wifivelocityd.md:4]:
```suggestion
> It should not be invoked manually.
```

File: pages/osx/wwand.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# wwand
+
+> USB WWAN device configuration daemon.
+> This should not be invoked manually.
```
Comment (navarroaxel) [inline pages/osx/wwand.md:4]:
```suggestion
> It should not be invoked manually.
```

File: pages/osx/xartstorageremoted.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+# xartstorageremoted
+
+> The xART Remote Storage Daemon. Receives save/fetch requests from the CoProcessor.
+> This should not be invoked manually.
```
Comment (navarroaxel) [inline pages/osx/xartstorageremoted.md:4]:
```suggestion
> It should not be invoked manually.
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**